cli.js

```diff
@@ -4,8 +4,10 @@ var errno = require('./')
   , arg   = process.argv[2]
   , data, code
 
-if (arg === undefined)
-  return console.log(JSON.stringify(errno.code, null, 2))
+if (arg === undefined) {
+  console.error(JSON.stringify(errno.code, null, 2));
+  process.exit(1);
```

Comment:
Why `console.error` and why `process.exit(1)`? This is a feature of the cli tool.